cherrypy/tutorial/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6-alpine
+
+RUN pip install cherrypy
+
+COPY ./ /tutorial
```

Comment:
Use `.` instead of `./`, it's more common

Reply:
Also done

Reply:
You forgot to push changes :)

Reply:
You are right! I will look quickly at the latest feedback and will push now